content/dstu2/authorization.md

```diff
@@ -35,54 +35,50 @@ Cerner currently supports the [SMART][4] launch workflow from within an EHR, suc
-* scope - Note: launch **must** be one of the scopes if using a [SMART][4] launch context. When requesting clinical data, [SMART scopes][11] for requesting clinical data must be provided.
+* scope - Note: launch **must** be one of the scopes if using a [SMART][4] launch context. When requesting clinical data, [SMART scopes][11] for requesting clinical data must be provided. At minimum, one scope must be provided in order for a successful request to occur.
 
 It is also **recommended** that your client application provides the following query parameters.
 
 * state - to prevent [cross-site request forgery attacks][2]
-* redirect_uri - note: The redirect_uri **must** match what was originally registered
+* redirect_uri - note: The redirect_uri **must** match what was originally registered and requested
 
-```
-https://authorization.sandboxcerner.com/realms/{TENANT_ID}/protocols/smart/authorize?response_type=code&client_id={YOUR_CLIENT_ID}&state=12345&redirect_uri=https%3A%2F%2Ftest.com%2Fcb
-```
+The authorization server authorize URL can be found in the security section of the [conformance document](conformance.md)
 
-If the user does not currently have an active session, the Authorization Server will redirect the user to the identity provider for the client organization in order to authenticate. 
+If the user does not currently have an active session, the Authorization Server will redirect the user to the identity provider for the client organization in order to authenticate.
 
 If successful, the authorization server will redirect the user-agent back to your client applications redirect_uri with a **code** query parameter. This is the authorization code that can be exchanged for an access token.
 
 ```
 https://test.com/cb?code=1234-567890ab-cdef
 ```
 
-Note: Using embedded browsers prevents single sign-on and authentication may fail for certain organizations who implement third party authentication systems. 
+Note: Using embedded browsers prevents single sign-on and authentication may fail for certain organizations who implement third party authentication systems.
 
 ### Requesting an access token ###
 Once an authorization code has been acquired, a back-channel **POST** to the authorization server's token endpoint can be made to request an access token. Following the [OAuth 2.0][1] spec, the authorization server accepts a content type of **application/x-www-form-urlencoded** from your client application with the following information.
 
-* grant_type
+* grant_type - currently only support "authorization_code"
 * code
 * client_id
-* redirect_uri (only if provided when the authorization code was requested)
+* redirect_uri (only necessary if it was provided when the authorization code was requested)
 
-```
-https://authorization.sandboxcerner.com/realms/{TENANT_ID}/protocols/smart/token
-```
+The authorization server authorize URL can be found in the security section of the [conformance document](conformance.md)
```

Comment:
You can `gem install nanoc` and run `nanoc` to generate the site and `nanoc view` to run it on localhost:3000

With (conformance.md) it was linking me to url/dstu2/authorization/conformance.md, with (../conformance) it was linking me to url/dstu2/conformance